doc/api/fs.md

```diff
@@ -2317,6 +2327,8 @@ changes:
     operation will ignore errors if you set this to false and the destination
     exists. Use the `errorOnExist` option to change this behavior.
     **Default:** `true`.
+  * `mode` {integer} modifiers for copy operation. **Default:** `0`.
+    See `mode` flag of [`fs.copyFile()`][]
```

Comment:
```suggestion
    See `mode` flag of [`fs.copyFile()`][].
```

Reply:
fixed in https://github.com/nodejs/node/pull/47084/commits/6a2b4bc7a566568ca65d5bcfc255a5761e424289